solutions/kattis-birthday.mdx

```diff
@@ -16,28 +16,32 @@ following definitions:
 
 - Define a person $x$ as a node.
 
-- If two nodes $a$ and $b$ have each others numbers, connect them with an
-  undirected edge.
+- If two nodes $a$ and $b$ have each other's numbers, connect them with an
+undirected edge.
 
 - Everyone can be invited to the party if there exists exactly one connected
-  component in the graph.
+component in the graph.
 
-Now the problem becomes: given a graph with $p (1 \leq p \leq 100)$ nodes and
-$c (0 \leq c \leq 5000)$ edges, can you remove some edge to break the graph into
-$\geq 1$ connected component?
+Now the problem becomes: given a graph with $p \: (1 \leq p \leq 100)$ nodes and
+$c \: (0 \leq c \leq 5000)$ edges, can you remove some edge to break the graph
+into more than one connected component?
 
 ## Applying DFS
 
-Since the constraints on edges (and nodes) is small, we can apply
-[dfs](/silver/dfs/). For each edge, let's run a DFS while ensuring we don't
-traverse that edge. If we can't visit some node, then the answer is "YES".
-Otherwise, if we can visit every node after trying to remove every edge, the
-answer is "NO" (notice the problem asks whether it's impossible to invite
-everyone). Check the code for more information on how to ignore an edge while
-DFS-ing.
+Since the constraints on edges (and nodes) is small, we can run
+[dfs](/silver/dfs/) on the graph. For each edge, let's run a DFS while ensuring
+we don't traverse that edge. If we can't visit some node, then the answer is
+"YES". Otherwise, if we're able to visit every node for each edge that gets
+removed, the answer is "NO" (notice the problem asks whether it's impossible to
+invite everyone). Check the code for more information on how to ignore an edge
+while DFS-ing.
```

Comment:
Since the condition seems to be the same for both of the implementations below, maybe it would be helpful to just say something like "we ignore an edge while dfsing if it's not in the adjacency matrix or it has been visited already," or remove it altogether, since this is fairly common knowledge.

Reply:
Since this is silver, I think it would probably be better to add an explanation like you mentioned just in case some people aren't super familiar with graphs